src/_data/plugins/eleventy-plugin-embedded-demos

```diff
@@ -0,0 +1,5 @@
+{
+  "npm": "eleventy-plugin-webmentions",
```

Comment:
is this right? doesn't seem to match the filename or description

Reply:
Fixed it, thanks!